crates/chain/src/tx_graph.rs

```diff
@@ -900,7 +900,7 @@ impl<A: Anchor> TxGraph<A> {
 )]
 #[must_use]
 pub struct Additions<A = ()> {
-    pub tx: BTreeSet<Transaction>,
+    pub txs: BTreeSet<Transaction>,
     pub txout: BTreeMap<OutPoint, TxOut>,
```

Comment:
I think you should also change `txout` to `txouts`

Reply:
ok... On it..

Reply:
Made the changes asked ... Please review @vladimirfomene